CHANGES_UNRELEASED.md

```diff
@@ -13,4 +13,12 @@
 
  - Android and iOS `Branch` objects no longer have access to a `FeatureConfig` object.
 
+### ⚠️ Breaking changes ⚠️
+- The experiment database will be migrating from version 1 to version 2 on
+  first run.  Various kinds of incorrectly specified feature and featureId
+  related fields will be detected, and any related experiments/enrollments will
+  be discarded.  If there is an error during the database upgrade, the database
+  will be wiped, since losing existing enrollments is still less bad than
+  having the database in an unknown inconsistent state.
+
```

Comment:
Is there an easy way to ensure that whenever a database gets wiped, an event gets sent to sentry?  That would be wonderful to monitor in Nightly and Beta as a way to be sure that we don't need to uplift anything.

Reply:
@jhugman I seem to recall you mentioning having context around how the errors are plumbed through outside of Rust to Sentry (or not) in the embedding apps.  Do you know the answer to this offhand?

Reply:
https://jira.mozilla.com/browse/SDK-287 filed for this.